internal/core/docker_host.go

```diff
@@ -19,13 +19,14 @@ type dockerHostContext string
 var DockerHostContextKey = dockerHostContext("docker_host")
 
 var (
-	ErrDockerHostNotSet               = errors.New("DOCKER_HOST is not set")
-	ErrDockerSocketOverrideNotSet     = errors.New("TESTCONTAINERS_DOCKER_SOCKET_OVERRIDE is not set")
-	ErrDockerSocketNotSetInContext    = errors.New("socket not set in context")
-	ErrDockerSocketNotSetInProperties = errors.New("socket not set in ~/.testcontainers.properties")
-	ErrNoUnixSchema                   = errors.New("URL schema is not unix")
-	ErrSocketNotFound                 = errors.New("socket not found")
-	ErrSocketNotFoundInPath           = errors.New("docker socket not found in " + DockerSocketPath)
+	ErrDockerHostNotSet                  = errors.New("DOCKER_HOST is not set")
```

Comment:
question: do these all need to be exported?

Reply:
Probably not. Given they are internal, I think we can rename all of them